app/models/papi_client.rb

```diff
@@ -358,6 +360,64 @@ def get_command_line_options(study_file, action)
     opts
   end
 
+  # set labels for pipeline request/virtual machine
+  #
+  # * *params*
+  #   - +action+ (String, Symbol) => action being executed
+  #   - +study+ (Study) => parent study of file
+  #   - +study_file+ (StudyFile) => File to be ingested/processed
+  #   - +user+ (User) => user requesting action
+  #   - +machine_type+ (String) => GCE machine type
+  #   - +boot_disk_size_gb+ (Integer) => size of boot disk, in GB
```

Comment:
Note to self: this is, indeed, documented as GB and not GiB [upstream](https://cloud.google.com/sdk/gcloud/reference/alpha/genomics/pipelines/run#--boot-disk-size).

Reply:
Yes - Google is not consistent in its use of nomenclature.  Billing uses GiB, but resource allocation is GB.  Totally not confusing at all.